cfme/configure/access_control/__init__.py

```diff
@@ -1007,7 +998,8 @@ class AllRolesView(ConfigurationView):
     def is_displayed(self):
         return (
             self.accordions.accesscontrol.is_opened and
-            self.title.text == 'Access Control Roles'
+            self.title.text == 'Access Control Roles' or
+            self.title.text == 'Role "{}"'.format(self.context['object'].name)
```

Comment:
Question: Why are you checking against a specific `Role` in the `AllRolesView`?

Reply:
Observations:
- If we created new role by navigating ```Roles > Click on any role > select copy this role to new role from configuration drop-down > update with new name and save``` (Here navigation goes to all page of roles - **self.title.text == 'Access Control Roles'**)
- Then go that ```new role > select Edit this role from configuration drop-down and update product features tree and save``` (Here navigation goes to details page of roles - **self.title.text == 'Role "{}"'.format(self.context['object'].name**)
- And if we tried to move to new role then it checks that if we are on ```AllRolesView```. Hence need to check ```is_diplayed``` for specific role.

Reply:
@ganeshhubale I believe this is a bug. I have seen something similar in condition creation. When you edit a role and hit save, you should be taken to the details page for that Role,  and the role name should be highlighted in the accordion.

What I'm seeing here is that when a role is edited and you hit save, the role name isn't highlighted in the accordion, which should be a BZ.